addon/content/new-tab-variation.js

```diff
@@ -160,8 +162,7 @@ function handleLoad(evt) {
   const win = evt.target.defaultView;
   const location = win.location.href;
   if (location === ABOUT_NEWTAB_URL || location === ABOUT_HOME_URL) {
-
-    Components.utils.import("resource://gre/modules/PrivateBrowsingUtils.jsm");
+    Cu.import("resource://gre/modules/PrivateBrowsingUtils.jsm");
```

Comment:
This solved the following new ESLint error:

```
/Users/pdehaan/dev/github/mozilla/tracking-protection-shield-study/addon/content/new-tab-variation.js
  164:5  error  Use Cu rather than Components.utils  mozilla/use-cc-etc
```